mqtt/mqtt-bridge/src/persist/loader.rs

```diff
@@ -201,8 +201,7 @@ mod tests {
         let mut state_lock = state.lock();
         let num_elements = 10_usize;
         for i in 0..num_elements {
-            #[allow(clippy::cast_possible_truncation)]
-            let key = Key { offset: i as u32 };
+            let key = Key { offset: i as u64 };
```

Comment:
offset is already u64, right?
```suggestion
            let key = Key { offset: i };
```